mnist/README.md

```diff
@@ -0,0 +1,24 @@
+# Extracting HLO from JAX MNIST example
```

Comment:
Let's assume we also have mnist from TF. Should we make the directory:
jax/mnist/
mnist/jax/
?
I don't really care either way but let's leave space for model AND framework.

Reply:
I like jax/mnist more, implemented